src/mlpack/tests/lmnn_test.cpp

```diff
@@ -293,61 +290,61 @@ BOOST_AUTO_TEST_CASE(LMNNSeparableEvaluateWithGradientTest)
 
   double objective = lmnnfn.EvaluateWithGradient(coordinates, 0, gradient, 1);
 
-  BOOST_REQUIRE_CLOSE(objective, 1.576, 1e-5);
+  REQUIRE(objective == Approx( 1.576).epsilon(1e-7));
```

Comment:
```suggestion
  REQUIRE(objective == Approx(1.576).epsilon(1e-7));
```

Remove the extra space.